Website/pages/mail.php

```diff
@@ -21,22 +21,12 @@ function sendVerificationMail($receiver, $person, $veritoken)
         $mail->SMTPSecure = PHPMailer::ENCRYPTION_STARTTLS;
         $mail->Port = 587;
 
-        $mail->setFrom(getenv("GMAILUSERNAME"), 'Datahunt team');;
+        $mail->setFrom('DatahuntRpg@gmail.com', 'Datahunt team');;
```

Comment:
is dit nodig?

Reply:
Deze aanpassing staat boven dat die outdated is, de code staat hier gewoon weer goed.